packages/web-frontend/src/components/ExportDialog.js

```diff
@@ -106,97 +165,117 @@ export const ExportDialog = ({
     setExpanded(!expanded);
   };
 
-  const loading = status === STATUS.EXPORTING || status === STATUS.ANIMATING;
+  const loading = status === STATUS.EXPORTING;
 
   return (
-    <StyledDialog open={isOpen} maxWidth="xs" fullWidth>
+    <StyledDialog open={isOpen} fullWidth>
       <CloseButton onClick={onClose}>
         <Close />
       </CloseButton>
       <Title>Export this chart</Title>
-      <FlexStart minHeight={160}>
-        {loading && (
-          <Box pb={3} style={{ margin: 'auto' }}>
-            <CircularProgress />
-          </Box>
-        )}
-        {status === STATUS.ERROR && (
-          <>
-            <StyledAlert severity="error">
-              There was an error with the export. Please try again.
-              <AlertAction onClick={() => onExport(selectedFormat)}>Retry export</AlertAction> or
-              contact <AlertLink href="mailto:support@tupaia.org">support@tupaia.org</AlertLink>
-            </StyledAlert>
-          </>
-        )}
-        {status === STATUS.IDLE && (
-          <div>
-            <Typography>The chart will be exported and downloaded to your browser</Typography>
-            <FormControl component="fieldset">
-              <StyledRadioGroup
-                name="format"
-                value={selectedFormat}
-                onChange={(e, value) => setSelectedFormat(value)}
-              >
-                {formats.map(format => (
-                  <FormControlLabel
-                    key={format}
-                    value={format}
-                    control={<Radio color="primary" />}
-                    label={formatLabels[format]}
-                  />
-                ))}
-              </StyledRadioGroup>
-            </FormControl>
-            {selectedFormat === 'png' && (
-              <>
-                <OptionsButton onClick={handleExpandClick}>
-                  <Typography>Advanced Options</Typography>
-                  <ExpandMore />
-                </OptionsButton>
-                <Collapse in={expanded} timeout="auto" unmountOnExit>
-                  <FormControlLabel
-                    control={
-                      <Checkbox
-                        color="primary"
-                        checked={exportWithLabels}
-                        onChange={event =>
-                          onChangeExportOptions({ exportWithLabels: event.target.checked })
-                        }
-                        name="exportWithLabels"
-                      />
-                    }
-                    label="Export With Labels"
-                  />
-                  {!exportWithTableDisabled && (
+      <ColContainer>
```

Comment:
Looks like a lot of changes but I just renamed some elements here, can use ?w=1 to see better